apps/cookbook/src/app/examples/accordion-example/accordion-example.component.html

```diff
@@ -1,19 +1,33 @@
-<h1>Accordion</h1>
+<div class="example-container">
+  <h1>Accordion</h1>
 
-<h2>Accordion</h2>
-<cookbook-accordion-default-example></cookbook-accordion-default-example>
+  <h2>Accordion</h2>
```

Comment:
Not related to your changes, but we can sync the headings so it does not say Accordion two times in a row 🙏 
```suggestion
  <h2>Simple Accordion</h2>
```